cmds.go

```diff
@@ -49,6 +72,75 @@ type SonyCmdToggle struct {
 	Input string `short:"i" help:"Specify host input, do not autodetect"`
 }
 
+// AfterApply creates a new [Screen] from the flags in the [screenFlags] struct.
+func (sf *screenFlags) AfterApply() error {
+	s, err := NewScreen(sf.Display, sf.Manufacturer, sf.ProductCode)
+	if err != nil {
+		return err
+	}
+	sf.screen = s
+	return nil
+}
+
+// Run (offscreen run) runs offscreen to turn the connected TV on and off
+// in line with X screen saver events.
+func (cmd *RunCmd) Run() error {
+	ch := make(chan bool)
+	g := errgroup.Group{}
+	g.Go(func() error { return cmd.screen.Watch(ch) })
+	g.Go(func() error { return cmd.loop(ch) })
+	return g.Wait()
+}
+
+func (cmd *RunCmd) loop(ch <-chan bool) error {
+	defer cmd.screen.Close()
+	c := NewRESTClient(cmd.Hostname, cmd.PSK)
+	ourLabel, err := hostnameLabel()
```

Comment:
`ourLabel` as opposed to `theirLabel`? as in current machine label?

Reply:
Yes, the label of the current machine. As opposed to the labels of the other inputs the TV has